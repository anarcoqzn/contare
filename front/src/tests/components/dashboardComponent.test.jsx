import DashboardComponent from '../../components/dasboardComponent/dashboardComponent';

it('should render DashboardComponent correctly', () => {
    const wrapper = shallow(<DashboardComponent />);
    expect(wrapper).toMatchSnapshot();
});

