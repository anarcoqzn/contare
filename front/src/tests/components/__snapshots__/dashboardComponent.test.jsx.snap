// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardComponent correctly 1`] = `<Route />`;
